pkgs/development/python-modules/opentelemetry-api/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, pythonPackages
```

Comment:
```suggestion
```
this is not allowed because it points to python2

Reply:
`pythonPackages` is overridden to be `self`: https://github.com/NixOS/nixpkgs/blob/d960ef5911882d64d3a57358a438597b061f9f42/pkgs/development/interpreters/python/python-packages-base.nix#L102

However, I'll be removing this reference to adopt your other suggestion.